recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "3.4.8" %}
+{% set version = "3.4.9" %}
 
 package:
   name: qgis
   version: {{ version }}
 
 source:
   url: https://qgis.org/downloads/qgis-{{ version }}.tar.bz2
-  sha256: fc98a207ec3d38acb8374f206dbcea666cd5249550da30e6e845ec1b8f4fbe8d
+  md5: 7b0839498e244a2e08353346d4c2edf6
```

Comment:
Let's use sha256 when possible. It is supposedly safer. See https://github.com/conda-forge/staged-recipes#2-how-do-i-populate-the-hash-field